src/mame/layout/h8.lay

```diff
@@ -31,10 +31,181 @@ license:CC0-1.0
+		<rect><bounds x="0.1" y="0.1" width="0.8" height="0.8" /><color red="0.15" green="0.15" blue="0.15" /></rect>
+		<text string="/"><bounds x="0.1" y="0.2" width="0.8" height="0.6" /><color red="1.0" green="1.0" blue="1.0" /></text>
+		<text string="ALTER"><bounds x="0.0" y="0.1" width="0.8" height="0.2" /><color red="1.0" green="1.0" blue="1.0" /></text>
+		<text string="RST 0"><bounds x="0.2" y="0.7" width="0.8" height="0.2" /><color red="1.0" green="1.0" blue="1.0" /></text>
+	</element>
+	<element name="btn_e">
+		<rect state="0"><bounds x="0.0" y="0.0" width="1.0" height="1.0" /><color red="0.2" green="0.2" blue="0.2" /></rect>
+		<rect state="1"><bounds x="0.0" y="0.0" width="1.0" height="1.0" /><color red="0.1" green="0.1" blue="0.1" /></rect>
+		<rect state="0"><bounds x="0.1" y="0.1" width="0.9" height="0.9" /><color red="0.1" green="0.1" blue="0.1" /></rect>
+		<rect state="1"><bounds x="0.1" y="0.1" width="0.9" height="0.9" /><color red="0.2" green="0.2" blue="0.2" /></rect>
+		<rect><bounds x="0.1" y="0.1" width="0.8" height="0.8" /><color red="0.15" green="0.15" blue="0.15" /></rect>
+		<text string="#"><bounds x="0.1" y="0.2" width="0.8" height="0.6" /><color red="1.0" green="1.0" blue="1.0" /></text>
+		<text string="MEM"><bounds x="0.0" y="0.1" width="0.8" height="0.2" /><color red="1.0" green="1.0" blue="1.0" /></text>
+		<text string="RTM 0"><bounds x="0.2" y="0.7" width="0.8" height="0.2" /><color red="1.0" green="1.0" blue="1.0" /></text>
+	</element>
+	<element name="btn_f">
+		<rect state="0"><bounds x="0.0" y="0.0" width="1.0" height="1.0" /><color red="0.2" green="0.2" blue="0.2" /></rect>
+		<rect state="1"><bounds x="0.0" y="0.0" width="1.0" height="1.0" /><color red="0.1" green="0.1" blue="0.1" /></rect>
+		<rect state="0"><bounds x="0.1" y="0.1" width="0.9" height="0.9" /><color red="0.1" green="0.1" blue="0.1" /></rect>
+		<rect state="1"><bounds x="0.1" y="0.1" width="0.9" height="0.9" /><color red="0.2" green="0.2" blue="0.2" /></rect>
+		<rect><bounds x="0.1" y="0.1" width="0.8" height="0.8" /><color red="0.15" green="0.15" blue="0.15" /></rect>
+		<text string="."><bounds x="0.1" y="0.1" width="0.8" height="0.6" /><color red="1.0" green="1.0" blue="1.0" /></text>
+		<text string="REG"><bounds x="0.0" y="0.1" width="0.8" height="0.2" /><color red="1.0" green="1.0" blue="1.0" /></text>
+	</element>
+
+	<group name="keypad">
+		<bounds x="0" y="0" width="4.35" height="4.275" />
+
+		<element ref="btn_7" inputtag="X0" inputmask="0x80"><bounds x="0.20" y="0.125" width="1.0" height="1.0" /></element>
+		<element ref="btn_8" inputtag="X1" inputmask="0x01"><bounds x="1.25" y="0.125" width="1.0" height="1.0" /></element>
+		<element ref="btn_9" inputtag="X1" inputmask="0x02"><bounds x="2.30" y="0.125" width="1.0" height="1.0" /></element>
+                <element ref="btn_a" inputtag="X1" inputmask="0x04"><bounds x="3.35" y="0.125" width="1.0" height="1.0" /></element>
```

Comment:
One of the lines here got indented more than the others.  Please use one tab per nesting level for indentation, and don’t indent using spaces.